buildstockbatch/schemas/v0.3.yaml

```diff
@@ -109,6 +109,7 @@ postprocessing-spec:
   partition_columns: list(str(), required=False)
   aws: include('aws-postprocessing-spec', required=False)
   keep_individual_timeseries: bool(required=False)
+  timestamps_uses_period_beginning: bool(required=False)
```

Comment:
Just to be clear, is this the proper input in the YAML?
```
postprocessing:
  timestamps_uses_period_beginning: true
  aws:
    ...
```

Reply:
Yes.